web/cobrands/sass/_base.scss

```diff
@@ -747,21 +755,44 @@ ul.error {
   }
 }
 
+html.mobile.js-nav-open #js-menu-open-modal {
+    position: absolute;
+    bottom: -1000px; // Just to be sure
+    left: 0;
+    right: 0;
+    background-color: rgba(0, 0, 0, 0.4);
+}
+
 // The nav list has this sort of "full width" look,
```

Comment:
I think if we change the `left: 1em; right: 1em;` below to `left: 0; right: 0;` then we can remove the `margin: 0 -1em;` and this comment too.

Reply:
Would mean we could _also_ drop a `margin: 0` from the `#main-nav` ruleset in `_layout.scss` too.